modules/integration/tests-common/admin-clients/src/main/java/org/wso2/am/admin/clients/mediation/SynapseConfigAdminClient.java

```diff
@@ -0,0 +1,222 @@
+     * @throws java.rmi.RemoteException throwable exception
+     */
+    public void activateService(String serviceName) throws RemoteException {
+        configServiceAdminStub.activate(serviceName);
+    }
+
+    /**
+     * Adding more configuration to the existing service
+     *
+     * @param serviceName - service name
+     * @throws java.rmi.RemoteException throwable exception
+     */
+    public void addExistingConfiguration(String serviceName) throws RemoteException {
+        configServiceAdminStub.addExistingConfiguration(serviceName);
+    }
+
+    /**
+     * @param serviceName - service name
+     * @param description - service description
+     * @throws java.rmi.RemoteException throwable exception
+     */
+    public void create(String serviceName, String description) throws RemoteException {
+        configServiceAdminStub.create(serviceName, description);
+    }
+
+    /**
+     * Deleting synapse configuration
+     *
+     * @param serviceName - service name
+     * @throws java.rmi.RemoteException throwable exception
+     */
+    public void deleteConfiguration(String serviceName) throws RemoteException {
+        configServiceAdminStub.deleteConfiguration(serviceName);
+    }
+
+    /**
+     * Get current synapse configuration
+     *
+     * @return synapse configuration
+     * @throws java.rmi.RemoteException throwable exception
+     */
+    public String getConfiguration() throws RemoteException {
+        return configServiceAdminStub.getConfiguration();
+    }
+
+    /**
+     * @return configuration list
+     * @throws java.rmi.RemoteException throwable exception
+     */
+    public ConfigurationInformation[] getConfigurationList() throws RemoteException {
+        return configServiceAdminStub.getConfigurationList();
+    }
+
+    /**
+     * save synapse configuration
+     *
+     * @throws java.rmi.RemoteException throwable exception
+     */
+    public void saveConfigurationToDisk() throws RemoteException {
+        configServiceAdminStub.saveConfigurationToDisk();
+    }
+
+    /**
+     * update synapse configuration
+     *
+     * @param configuration - synapse configuration
+     * @return configuration update status
+     * @throws java.rmi.RemoteException            throwable exception
+     * @throws javax.servlet.ServletException      throwable exception
+     * @throws javax.xml.stream.XMLStreamException throwable exception
+     */
+    public boolean updateConfiguration(String configuration)
+            throws XMLStreamException, ServletException, RemoteException {
+        return configServiceAdminStub.updateConfiguration(createOMElement(configuration));
+    }
+
+    /**
+     * Uploads synapse config bu using a file
+     *
+     * @param file -File that contains the synapse configuration
+     * @return
+     */
+    public boolean updateConfiguration(File file)
+            throws IOException, SAXException, ParserConfigurationException, TransformerException,
```

Comment:
This is a admin client and we need to throw  the original excepting to users. As per the offline disscusstion we do not use a custom exception here